src/main/java/com/gotech/accesscontrol/constant/UserType.java

```diff
@@ -0,0 +1,11 @@
+package com.gotech.accesscontrol.constant;
```

Comment:
Add constructor in enum which takes String value
Add public String getValue() return value